about.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About us</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"/>
+  <link rel="stylesheet" type="text/css" href="./assets/css/style.css">
+</head>
+<body>
+  <div class="section">
+    <div class="container">
+      <header>
+        <!-- Nav bar -->
+        <nav class="navbar navbar-expand-lg navbar-light bg-danger custom-nav">
```

Comment:
I really like the logo - thank you!  Please could you set the background colour of the navbar to the one from our colour palette #9A031E.  Also I think the navbar should take up the full width of the page in the way the home page navbar does?  At the moment there's white space to the left and right of it on the about page.